cmd/pulumi-test-language/test_host.go

```diff
@@ -122,33 +120,6 @@ func (h *testHost) LanguageRuntime(runtime string, info plugin.ProgramInfo) (plu
 }
 
 func (h *testHost) EnsurePlugins(plugins []workspace.PluginSpec, kinds plugin.Flags) error {
-	// EnsurePlugins will be called with the result of GetRequiredPlugins, so we can use this to check
```

Comment:
This is tested in the main framework function now, so isn't needed here and the simple code here is wrong when dealing with MLCs